### DISCUSSION TITLE
How to run in windows

### DISCUSSION THREAD

**Discussion by rcraig114** [ep1cman/unifi-protect-backup#36] [General]
This is probably a stupid question, but how do I run this from windows? I'm able to install the requirements, but actually running the tool errors out. What am I missing?

pip unifi-protect-backup -v --address 10.253.X.X --no-verify-ssl --username 'unifi-protect-XXXXXX' --password 'XXXXXX' --rclone-destination XXXX-onedrive:/Unifi-recordings/
ERROR: unknown command "unifi-protect-backup"

**Comment by ep1cman** [comment]
Sorry I don't really use windows much, but it you shouldn't be putting `pip` at the start of that. you can either call `unifi-protect-backup` if python has been install to be included in your `PATH` variable. Or you can run `python unifi-protect-backup/cli.py {arguments}`

**Reply by rcraig114** [reply]
OK, gotcha. I ended up spinning up a Ubuntu box to run it on. Any pros/cons to running this script directly on the Cloud Key Gen 2?

**Reply by ep1cman** [reply]
So long as it persists between updates I would say it's a very sensible place for it to run tbh
